backend/day-01/test.py

```diff
@@ -1,6 +1,13 @@
-name = None
-age = None
-isMale = None
-department = None
+# OPTION 1
+# my_name = "Harvy Jones Pontillas";
+# name_of_department = "DILG";
+# about_me = "I drink rice"
 
-print(f"{name} is an {age}-year old {'male' if isMale else 'female'} participating in {department}'s 30 Days of Coding.")
\ No newline at end of file
+# OPTION 2
+my_info = {
+    'name': "Harvy Jones Pontillas",
+    'name_of_department': "DILG",
+    'about_me': "I love drinking rice",
+};
+
+print(f'Hello, I am {my_info["name"]}. I love {my_info["name_of_department"]} and I love being in the Backend even more! One thing about me is that {my_info["about_me"]}.');
```

Comment:
This line seems to be too long. Perhaps write it in a multiline format for it to be more readable 👍

Reply:
ulolll